### DISCUSSION TITLE
Using the Visual Tests Addon with Storybook Composition

### DISCUSSION THREAD

**Discussion by thafryer** [chromaui/addon-visual-tests#321] [Ideas]
Can I use Storybook Visual Test Addon on with Storybook's Composition? A user has 3 Storybooks, using composition, but when they click the "play" button of the addon, stories are only tested in the main one, not the composed one. I'd like to, when I click in the "play", all of them being processed and the results are shown to me in a single project in Chromatic.

An example use case is a Turborepo monorepo, consisting of internal libraries being consumed by a Next.js app.

**Comment by shilman** [comment]
@thafryer Is the expectation that they have already set up chromatic projects for the main storybook and each of the composed storybooks, so pressing the play button would kick off separate builds and aggregate the results in the UI?

**Reply by thafryer** [reply]
From what I'm understanding looking at the Support ticket, this seems to be correct!
